supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc

```diff
@@ -11,37 +11,38 @@
 
 
 [discrete]
-[[ibm-eserver-system-p]]
-==== image:images/yes.png[yes] IBM eServer System p (noun)
-*Description*: Use "IBM eServer System p" for the first reference, and "IBM System p" or "System p" for subsequent references.
+[[ibm-power]]
+==== image:images/yes.png[yes] IBM Power® (noun)
+*Description*: _IBM Power_ represents the family of servers based on Power processors that can include combinations of IBM Power servers and its products.
 
 *Use it*: yes
 
-*Incorrect forms*: pSeries
+*Incorrect forms*: pSeries, System P, Power Systems
 
 *See also*:
 
 [discrete]
-[[ibm-s-390]]
-==== image:images/yes.png[yes] IBM S/390 (noun)
-*Description*: The _IBM S/390_ is the IBM large server (or mainframe) line of computer systems. Use the full description "IBM S/390".
+[[ibm-linuxone]]
+==== image:images/yes.png[yes] IBM® LinuxONE (noun)
+*Description*: _IBM® LinuxONE_ represents the IBM® LinuxONE platform (`s390x` architecture) and its
+products.
 
 *Use it*: yes
 
-*Incorrect forms*: S/390, S90
+*Incorrect forms*: LinuxONE, linuxone
 
-*See also*:
+*See also*: xref:ibm-z[IBM Z]
 
 [discrete]
 [[ibm-z]]
-==== image:images/yes.png[yes] IBM Z (noun)
-*Description*: _IBM Z_ is the new name for the "IBM z Systems" family of IBM mainframe computers.
+==== image:images/yes.png[yes] IBM Z® (noun)
```

Comment:
```suggestion
==== image:images/yes.png[yes] IBM Z (noun)
```
Remove the trademark - we don't want writers to use the trademark in product docs.

Reply:
I am now noticing there are trademark icons on several terms. We don't want to include the trademark unless we want writers to use the trademark in documentation - these glossary entries are instructing writers on how to spell these terms correctly, including the trademark icon. Please review the list of terms and make sure trademark icons are included only when that applies (if it ever does? Do we ever want writers to use trademark icons in product docs?).

Reply:
Hi @mportman12 . In the OCP docs we do use trademark icons on IBM components. See https://docs.openshift.com/container-platform/4.13/installing/installing_ibm_z/preparing-to-install-on-ibm-z.html . Usually on first reference of an IBM technology name.

Reply:
For IBM® LinuxONE there's this guidance to always use the trademark with LinuxONE: 

`Effective November 15, 2022, IBM® must precede LinuxONE in all situations`

Reply:
@lahinson ^^ regarding https://github.com/openshift/openshift-docs/pull/61782#discussion_r1265438957